src/gui/mzroll/tabledockwidget.cpp

```diff
@@ -812,6 +878,13 @@ void TableDockWidget::exportJson() {
 }
 
 
+void TableDockWidget::exportPolly() {
+    LOGD;
+    _pollyIntegration = new PollyIntegration(this);
```

Comment:
Object is not handle properly.

Reply:
What do you mean by "not handled properly" ??
Please be more specific